src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -365,6 +365,173 @@ public async Task PostAsync_StreamRefused_RequestIsRetried()
             }
         }
 
+        [ConditionalFact(nameof(SupportsAlpn))]
```

Comment:
Since these tests are dependent on delays, they should be Outerloop.

Reply:
Fixed